docs/usage/key-concepts/how-renovate-works.md

```diff
@@ -0,0 +1,164 @@
+---
+title: How Renovate works
+description: Learn how Renovate works
+---
+
+# Introduction
+
+Renovate usually performs these steps:
+
+- Clones the repo
+- Scans package files to extract dependencies
+- Looks up registries to check for available updates
+- Applies any grouping rules defined
+- Pushes branches and raises Pull Requests
+
+Because Renovate needs to support a lot of dependency naming and versioning conventions, it has modules for each known convention.
+You can contribute your own modules, if you want.
+
+## Modules
+
+Renovate's modules are:
+
+- [datasource](../modules/datasource/index.md)
+- [manager](../modules/manager/index.md)
+- [platform](../modules/platform/index.md)
+- [versioning](../modules/versioning.md).
```

Comment:
@HonkingGoose @viceice should these drop the index.md so that they work in the online docs?

Reply:
@rarkins I don't know, I usually mess up the links the first time round. I hope viceice knows. 🙂

Reply:
There are multiple cases where the link to `index.md` is done, and it works well

https://github.com/renovatebot/renovate/blob/main/docs/usage/user-stories/maintaining-aur-packages-with-renovate.md?plain=1#L36

https://docs.renovatebot.com/user-stories/maintaining-aur-packages-with-renovate/

Reply:
the link needs to point to the markdown file to not get warnings from mkdocs